### PR TITLE
R81

### PR DESCRIPTION
**Pull request by fblundun**

### CODE REVIEW

**Comment by alexanderdean** [comment]
Thanks @fblundun ! Can you retitle the commits for #2692 and #2694 to match the ticket names?

**Comment by fblundun** [comment]
Done!

File: 3-enrich/hadoop-event-recovery/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Usage
+
+Run this job using the [AWS Command Line Interface] [awscli]:
+
+```
+aws emr create-cluster --applications Name=Hadoop --ec2-attributes '{
+    "InstanceProfile":"EMR_EC2_DefaultRole",
+    "AvailabilityZone":"us-east-1d",
+    "EmrManagedSlaveSecurityGroup":"sg-2f9aba4b",
+    "EmrManagedMasterSecurityGroup":"sg-2e9aba4a"
```
Comment (alexanderdean) [inline 3-enrich/hadoop-event-recovery/README.md]: Replace these with {{}}s

File: 3-enrich/hadoop-event-recovery/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Snowplow Hadoop Event Recovery
+
+## Introduction
+
+Use this Scalding job to extract raw Snowplow events from your Snowplow bad rows JSONs and fix any problems with them, making them ready for reprocessing.
+
+## Usage
```
Comment (alexanderdean) [inline 3-enrich/hadoop-event-recovery/README.md]: All of this documentation should be moved into a dedicated wiki page...

File: 3-enrich/hadoop-event-recovery/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# Snowplow Hadoop Event Recovery
```
Comment (alexanderdean) [inline 3-enrich/hadoop-event-recovery/README.md]: This readme should primarily push users to the rich wiki documentation

**Pull request merged**